### ISSUE TITLE
404 GET /nbextensions/qgridjs/qgrid.widget.js

### ISSUE THREAD

**Issue by hzzyyy** [quantopian/qgrid#92]
I used conda to instal qgrid: 
`conda install qgrid`
Won't work properly with jupyter-notebook

It seemed the wrong path was searched for qgrid.widget.js. 
When conda installed this package, the javascipts are located under:
$(CondaDir)/envs/envname/lib/site-packages/qgrid/qgridjs
I tried to add this to the 'c.NotebookApp.extra_nbextensions_path' of '~/.jupyter/jupyter_notebook_config.py' but it didn't work.

I haven't tried pip to install qgrid, and would like to confirm that conda-installed qgrid cannot work before moving to pip.

It might be an issue on conda side. I hope this ticket can bring some good discussion.

**Comment by hzzyyy** [comment]
qgrid: 0.3.2
ipython 5.1.0
jupyter-notebook 4.2.3

**Comment by mickeydonkey** [comment]
Hi @hzzyyy same problem on win10, rolling back to qgrid 0.3.1 helps me address it temporarily.

**Comment by mickeydonkey** [comment]
@hzzyyy Try run first `qgrid.nbinstall(overwrite=True) ` after installation.

**Comment by darribas** [comment]
I have a similar problem:

```
[W 09:53:45.284 NotebookApp] 404 GET /nbextensions/widgets/widgets/js/widget.js?v=20161125095255 (127.0.0.1) 1.05ms referer=http://localhost:8888/notebooks/Untitled.ipynb?kernel_name=python3
```

This is on a linux box and an environment with Python 3.5 and:

```
(gds3) [~/Desktop] $ conda list | grep qgrid
qgrid                     0.3.2                    py35_0  
(gds3) [~/Desktop] $ conda list | grep ipython
ipython                   5.1.0                    py35_1    conda-forge
ipython_genutils          0.1.0                    py35_0    conda-forge
(gds3) [~/Desktop] $ conda list | grep ipywidgets
ipywidgets                5.2.2                    py35_2    conda-forge
(gds3) [~/Desktop] $ conda list | grep notebook
notebook                  4.2.3                    py35_0    conda-forge
(gds3) [~/Desktop] $ conda list | grep jupyter
jupyter                   1.0.0                    py35_3  
jupyter_client            4.4.0                    py35_0    conda-forge
jupyter_console           5.0.0                    py35_0    conda-forge
jupyter_core              4.2.0                    py35_0    conda-forge
(gds3) [~/Desktop] $ 
```

When I try to `qgrid.show_grid`, nothing happens. No error, but no display either. Any suggestions? Thanks!

**Comment by jonschull** [comment]
me too.  But a solution...
Problem:
I followed the recipe at http://nbviewer.jupyter.org/gist/TimShawver/8fcef51dd3c222ed25306c002ab89b60

and got this 

qgrid.show_grid(spy)
Widget Javascript not detected.  It may not be installed properly. Did you enable the widgetsnbextension? If not, then run "jupyter nbextension enable --py --sys-prefix widgetsnbextension"

running ...
jupyter nbextension enable --py --sys-prefix widgetsnbextension 
...did not fix the problem

Then I installed from source and  restarting the jupyter server 
WORKED.

**Comment by kellington** [comment]
I have this problem as well - the import and the nbinstall work no errors - but qgrid.show_grid shows nothing.
Might it have something to do with virtualenv?   
When qgrid.nbinstall is run, what files are supposed to be copied where?
[pip_installed.txt](https://github.com/quantopian/qgrid/files/959877/pip_installed.txt)

**Comment by leoalekseyev** [comment]
This is broken for me as well; js console shows that the frontend is attempting (and failing) to fetch `http://localhost:8888/nbextensions/widgets/widgets/js/widget.js`.  Note that this is different than the route in the original post.  I see that `qgrid.nbinstall` puts some javascript into `/home/leo/.local/share/jupyter/nbextensions/qgridjs`, but I don't see any requests for it.  WTF?

**Comment by jsinglet** [comment]
I am having the 404 problem after installing from conda. Any word on a fix?

**Comment by leoalekseyev** [comment]
@jsinglet See my comment above and try the workaround in the discussion for issue #109 or build it from my fork (see PR #112)

**Comment by TimShawver** [comment]
Assuming everyone was experiencing the same issue that @leoalekseyev found with qgrid not being compatible with jupyter notebook 5.x, this is now fixed in the latest release, v0.3.3.  Thanks for reporting and sorry about the inconvenience.  Please feel free to open a new issue or reply here if you're still having a problem.
